core/settings/page/manager.php

```diff
@@ -173,7 +173,7 @@ protected function print_editor_template_content( $name ) {
 			<# } ); #>
 		</div>
 		<# } #>
-		<div id="elementor-panel-<?php echo $name; ?>-settings-controls"></div>
+		<div id="elementor-panel-<?php echo esc_attr( $name ); ?>-settings-controls"></div>
```

Comment:
"the variable $name does not contain a user input value"

Reply:
Same as above - better to remove?

Reply:
@matipojo - as agreed, keeping the esc_attr for not adding the comment again.